app/lib/reports/offline_session_exporter.rb

```diff
@@ -206,7 +216,9 @@ def rows(patient_session:)
       }
 
       vaccination_records =
-        patient.vaccination_records.select { it.programme_id == programme.id }
+        patient.vaccination_records.to_a.select do
```

Comment:
I'm wondering what the `to_a` adds here? I would expect `select` to have already called it.